notebooks/01_collect_data/raw_to_text.ipynb

```diff
@@ -17,28 +17,27 @@
   },
```

Comment:
<p>I know you're just moving code around, but why not to take a look at it. :D</p><p> </p><p>Maybe not directly useful in this particular case:</p><p> </p><p>Python has a built in <code>calendar</code> module, so you can get <a href="https://docs.python.org/3/library/calendar.html#calendar.day_name" target="_blank">month and weekday names</a> without hard-coding stuff</p><p> </p><pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; import calendar

&gt;&gt;&gt; [i for i in calendar.month_name]
['', 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']

&gt;&gt;&gt; [i for i in calendar.month_abbr]
['', 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']

&gt;&gt;&gt; [i for i in calendar.day_name]
['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']

&gt;&gt;&gt; [i for i in calendar.day_abbr]
['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
</pre><p> </p><p>I'd also suggest to make the regex checks case insensitive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>